packages/rum/test/viewCollection.spec.ts

```diff
@@ -308,6 +308,71 @@ describe('rum view measures', () => {
     jasmine.clock().uninstall()
   })
 
+  it('should have a loadEventEnd loading time when having no activity', () => {
```

Comment:
no, for initial view, it should use loadEvent when it is bigger than loading time